docker-compose.yml

```diff
@@ -1,10 +1,19 @@
 version: '3.3'
```

Comment:
```suggestion
version: '3.5'
```

Reply:
**3.5** works for me, just not **3.6** :frowning_face:

Reply:
I'm gonna leave it at 3.3 now. I bumped it up to try and solve a problem with .env in the front-end project. Which it didn't.